rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -73,13 +75,12 @@ def add_init_obs(self, episode_id: EpisodeID, agent_index: int,
             self._build_buffers(
                 single_row={
                     SampleBatch.OBS: init_obs,
-                    SampleBatch.EPS_ID: episode_id,
                     SampleBatch.AGENT_INDEX: agent_index,
                     "env_id": env_id,
                     "t": t,
                 })
         self.buffers[SampleBatch.OBS].append(init_obs)
-        self.buffers[SampleBatch.EPS_ID].append(episode_id)
+        self.episode_id = episode_id
```

Comment:
Don't have to "collect" these. They are always the same for the same agent anyways.